src/dswx_sar/metadata.py

```diff
@@ -266,24 +337,54 @@ def _get_general_dswx_metadata_dict(cfg, product_version=None):
 
     # save datetime 'YYYY-MM-DDTHH:MM:SSZ'
     dswx_metadata_dict['PROCESSING_DATETIME'] = datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
-    dswx_metadata_dict['POLARIZATION'] = cfg.groups.processing.polarizations
 
     return dswx_metadata_dict
 
 
-def gather_rtc_files(rtc_dirs):
+def gather_rtc_files(rtc_dirs, pol):
     """
     Given directories containing RTC files, gather all h5 files.
```

Comment:
```suggestion
    Given directories containing RTC files, gather all TIF files of the polarization `pol`.
    
    Parameters
    ------------
    rtc_dirs: str
        RTC output directory
    pol: str:
        polarization
    
    Returns:
    ---------
    tif_files: list
        List of RTC output TIF files in the polarization `pol`
```

Did not check the indentation in the suggestion above.